apps/api/src/hyp3_api/routes.py

```diff
@@ -93,67 +92,57 @@ def default(self, o):
         json.JSONEncoder.default(self, o)
 
 
-class NonValidator:
-    def __init__(self, spec):
-        pass
-
-    def validate(self, res):
-        return ResponseValidationResult()
-
-
 class ErrorHandler(FlaskOpenAPIErrorsHandler):
     def __init__(self):
         super().__init__()
 
-    @classmethod
-    def handle(cls, errors):
-        response = super().handle(errors)
+    def __call__(self, errors):
+        response = super().__call__(errors)
         error = response.json['errors'][0]
         return handlers.problem_format(error['status'], error['title'])
 
 
-class Jobs(FlaskOpenAPIView):
-    def __init__(self, spec):
-        super().__init__(spec)
-        self.response_validator = NonValidator
-        self.openapi_errors_handler = ErrorHandler
+app.json_encoder = CustomEncoder
+
+openapi = FlaskOpenAPIViewDecorator(
+    api_spec,
+    response_cls=None,
+    errors_handler_cls=ErrorHandler,
+)
+
 
-    def post(self):
-        return jsonify(handlers.post_jobs(request.get_json(), g.user))
+@app.route('/jobs', methods=['POST'])
+@openapi
+def jobs_post():
+    return jsonify(handlers.post_jobs(request.get_json(), g.user))
```

Comment:
This is completely nitpicky but I like the methods in alphabetical order for a route

Reply:
Fair, I'm just going to merge since I'm eager to get this deployed, but will keep in mind for the future.